.github/workflows/release-binaries.yml

```
@@ -45,8 +45,14 @@ jobs:
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
       - run: |
           nix-build -A utils.release.${{ matrix.target }}.kubectl-plugin --arg incremental false ${{ matrix.system }}
+      - name: Tar files
+        run: |
+          out_dir="$(pwd)"
+          pushd result/bin
+          tar -czvf "${out_dir}/kubectl-mayastor-${{ matrix.arch }}-${{ matrix.target }}.tar.gz" kubectl-mayastor${{ matrix.suffix}}
+          popd
```

nit: maybe we could use the `-C` option here to avoid changing workspace directory?
```suggestion
          tar -czvf "kubectl-mayastor-${{ matrix.arch }}-${{ matrix.target }}.tar.gz" -C result/bin kubectl-mayastor${{ matrix.suffix}}
```